modelerfour/test/scenarios/expected/head/modeler.yaml

```diff
@@ -82,11 +82,11 @@ operationGroups:
             language: !<!Languages> 
               default:
                 name: ''
-                description: ''
+                description: Upload successful
             protocol: !<!Protocols> 
               http: !<!HttpResponse> 
                 statusCodes:
-                  - '200'
+                  - '204'
```

Comment:
Weird, why did the status code change?

Reply:
I changed it in the oai3 file to be consistent with the payload not returning any content